apps/client/src/shared/utils/ogImage.ts

```
@@ -0,0 +1,138 @@
+      break; // 첫 성공시 루프 종료
+    } catch (err) {
+      lastError = err;
+      continue;
+    }
+  }
+  if (!response) {
+    throw lastError || new Error('모든 프록시 서비스에서 실패했습니다.');
+  }
+
+  const ogData = parseOGData(response.data);
+  // 이미지: 상대경로일 경우 보정
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)